portfolio/src/main/webapp/index.html

```diff
@@ -133,7 +134,21 @@ <h1>Andrzej Podobinski</h1>
             </div>
            </div>
       </div>
+      <div>
+        <div id="comments-container">
+            <ul id="comments" action="/data" method="POST">
+            </ul>
+        </div>
+        <form action="/text" method="POST">
```

Comment:
Post is not implemented yet right? Perhaps instead of having user get a 404 page, it's worth returning a very simple "Not Implemented" response to the user. It's OK not to add in this PR if you're already implementing it in later PRs, but worth keeping in mind when working on the capstone project.

Reply:
it actually is but on server-side branch, I was developing them separately. But it's good advice to implement such simple responses, thanks.